### PR TITLE
Feat/project stack

### PR DESCRIPTION
**Pull request by CuriousCoder00**
Fixes: #522 #526 

https://github.com/user-attachments/assets/97d3026f-b562-42bf-9f83-7856f746e070

### CODE REVIEW

**Comment by CuriousCoder00** [comment]
@VineeTagarwaL-code review please
and may i start working on #522 in this pr?

**Comment by CuriousCoder00** [comment]
- Added Project thumbnail
- Refactored code to improve project showcase ui
- Refactored code to mention the issue #526 
https://github.com/user-attachments/assets/60fc1960-6a43-490a-a0bb-02228605cd1d

**Comment by CuriousCoder00** [comment]
@VineeTagarwaL-code please review the PR and tell me if there are any improvements/changes needed

**Comment by VineeTagarwaL-code** [comment]
Lessss goooooooo !!!!!

**Comment by VineeTagarwaL-code** [comment]
Loved the showcase, reviewing give me a minute

File: prisma/schema.prisma [hunk 1]
```diff
@@ -80,32 +80,43 @@ model Job {
...
+  projectGithub    String
+  stack            ProjectStack @default(OTHERS)
+  userId           String
+  user             User         @relation(fields: [userId], references: [id], onDelete: Cascade)
+}
+
+enum ProjectStack {
+  GO
+  PYTHON
+  MERN_NEXTJS
```
Comment (VineeTagarwaL-code) [inline prisma/schema.prisma:116]: dont mix mern with nextjs, have them as seperate, dont add java springboot will be a good idea in replacement of java

File: src/components/profile/UserProject.tsx [hunk 2]
```diff
@@ -43,35 +45,58 @@ export function UserProjects() {
...
+                  <SquareArrowOutUpRightIcon size={16} />
+                </Link>
+              )}
               <Link
-                href={item.projectLiveLink}
+                href={item.projectGithub}
                 className="text-blue-500 hover:underline"
               >
-                Live Project
+                <LucideGithub size={16} />
```
Comment (VineeTagarwaL-code) [inline src/components/profile/UserProject.tsx:97]: use the icon file we have, much clean <icon.github/>

**Review by VineeTagarwaL-code — CHANGES_REQUESTED** [review]

**Comment by CuriousCoder00** [comment]
@VineeTagarwaL-code Got it.. making the changes in few mins

**Comment by CuriousCoder00** [comment]
image
Made the required changes

**Comment by CuriousCoder00** [comment]
Refactored code for experience showcase
Before:
image


After:
image

**Comment by CuriousCoder00** [comment]
- Implemented Add more button in Projects, Experiences, Skills


Should i add extra functionalities such as:
- Allowing user to edit projects/experiences/skillsets details
- Allowing user to delete as well

File: prisma/schema.prisma (hunk 1 above)
Comment (CuriousCoder00) [reply prisma/schema.prisma]: made changes for ProjectStack enums as well as wherever it's getting used

**Review by VineeTagarwaL-code — APPROVED** [review]

**Comment by VineeTagarwaL-code** [comment]
Noice work man, do dm me in discord once i will put you up in bounty list

also make sure you post on x to catch good eyes of recruiters, OS helps a lot ! <3

**Comment by CuriousCoder00** [comment]
> Noice work man, do dm me in discord once i will put you up in bounty list
> 
> also make sure you post on x to catch good eyes of recruiters, OS helps a lot ! <3

@VineeTagarwaL-code 
I appriciate your kind words, i'll dm you on discord soon and will tag you and kirat bhaiya when i post on x

**Pull request merged**